core/src/main/scala/org/apache/spark/sql/delta/OptimisticTransaction.scala

```diff
@@ -1128,6 +1136,7 @@ trait OptimisticTransactionImpl extends TransactionalWrite
             assertMetadata(m)
           case p: Protocol =>
             recordProtocolChanges(snapshot.protocol, p, isCreatingNewTable)
+          case _: DomainMetadata => numOfDomainMetadatas += 1
```

Comment:
nit: all the other cases in this match have a newline after the `=>`;
should we match style here for consistency?

Reply:
fixed